trivy.yaml

```diff
@@ -0,0 +1,14 @@
+# https://aquasecurity.github.io/trivy/latest/docs/references/configuration/config-file/
+---
+# Specify an exact list of recognized and acceptable licenses.
+# [A GitHub workflow](/.github/workflows/trivy.yaml) rejects pull requests that
+# import licenses not in this list.
+#
+# https://aquasecurity.github.io/trivy/latest/docs/scanner/license/
+license:
+  ignored:
```

Comment:
So right now we're telling Trivy to ignore these licenses; if another license shows up that's not in this list, we'll get an alert because the job is configured with `--exit-code=1`: any license at any level -- `Forbidden`, `Restricted`, `Unencumbered`, et al. -- will alert if it is not ignored, right?

Reply:
Right. Every license that Trivy recognizes has a severity, and this job alerts on all severities.

We'll also be alerted if Trivy can't detect the license. Those get a severity of `UNKNOWN`.